### PR TITLE
feat: add newsletter registration and confirmation

### PR DESCRIPTION
**Pull request by bfabio**

### CODE REVIEW

**Comment by developers-italia-droid** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  There is a problem with the commit message
> wip: newsletter
- type must be one of [build, chore, ci, docs, feat, fix, perf, refactor, revert, style, test]
  </td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against bcb4d5d4f2ad17bde068d1fae1107e8e3742dc9d
</p>

**Comment by developers-italia-droid** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>There is a problem with the commit message
> Update privacy-policy.md
- subject may not be empty
- type may not be empty</td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>There is a problem with the commit message
> Update privacy-policy.md
- subject may not be empty
- type may not be empty</td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 015c8e72d0d8e897eb078e8d452930e31f1028bd
</p>

**Review by sebbalex — APPROVED** [review]

File: _data/l10n.yml [hunk 1]
```diff
@@ -131,7 +131,8 @@ it:
     home_news_expand: per saperne di più
     home_tags: Cerca per categoria
     home_tags_more: Altri filtri
-    newsletter_text: Iscriviti alla mailing list per ricevere aggiornamenti e novità. È una list a basso traffico (promesso!) che ti aiuterà a rimanere in contatto con noi.
+    # newsletter_text: Iscriviti alla mailing list per ricevere aggiornamenti e novità. È una list a basso traffico (promesso!) che ti aiuterà a rimanere in contatto con noi.
```
Comment (sebbalex) [inline _data/l10n.yml:134]:
```suggestion
```

File: Gemfile.lock [hunk 2]
```diff
@@ -132,7 +132,7 @@ DEPENDENCIES
   tzinfo-data
 
 RUBY VERSION
-   ruby 2.7.3p183
+   ruby 2.6.9p207
```
Comment (sebbalex) [inline Gemfile.lock:135]: any specific reason for this?

**Review by sebbalex — APPROVED** [review]

**Comment by bfabio** [comment]
@sebbalex ready to go :+1:

**Comment by sebbalex** [comment]
LGTM

**Pull request merged**